### ISSUE TITLE
(Claude) Error: all messages must have non-empty content

### ISSUE THREAD

**Issue by frmsaul** [karthink/gptel#425]
Sometimes i get:

`Claude error: (HTTP/2 400) messages.4: all messages must have non-empty content except for the optional final assistant message` 

error with gptel-send even though i selected a region.

It's unclear when it works and when it doesn't. For instance: 

It broke on: 
```
// use disabled
function ViewProfileButton({ viewingProfile, setViewingProfile, disabled }) {
  return (
    <div
      className="flex items-center justify-center text-sm text-blue-500 mb-2 cursor-pointer"
      onClick={() => setViewingProfile(!viewingProfile)}
    >
      <span>{viewingProfile ? "Hide profile" : "View profile"}</span>
      <ChevronUp
        className={`w-4 h-4 ml-1 transition-transform duration-300 ${
          viewingProfile ? "transform rotate-180" : ""
        }`}
      />
    </div>
  );
}
```

But then worked on 
```
// use disabled

\`\`\`jsx
function ViewProfileButton({ viewingProfile, setViewingProfile, disabled }) {
  return (
    <div
      className="flex items-center justify-center text-sm text-blue-500 mb-2 cursor-pointer"
      onClick={() => setViewingProfile(!viewingProfile)}
    >
      <span>{viewingProfile ? "Hide profile" : "View profile"}</span>
      <ChevronUp
        className={`w-4 h-4 ml-1 transition-transform duration-300 ${
          viewingProfile ? "transform rotate-180" : ""
        }`}
      />
    </div>
  );
}
\`\`\`
```

**Comment by karthink** [comment]
If you're not on the latest commit, please update and try.

**Comment by karthink** [comment]
Closing since this is fixed in #409.  Please use that thread (reopening it if necessary) if the problem still persists.
